uptime_service_validation/coordinator/helper.py

```diff
@@ -367,3 +367,10 @@ def getExistingNodes(conn, logger):
     finally:
         cursor.close()
     return nodes
+
+
+def sendSlackMessage(url, message, logger):
```

Comment:
I believe we should stick to python naming convention for functions which is `snake_case`.

Reply:
What I have is consistent with the other functions in the helper, so I'll leave it as is.